code/game/machinery/atmoalter/scrubber.dm

```diff
@@ -155,7 +155,7 @@
 	active_power_usage = 100000	//100 kW ~ 135 HP
 
 	var/global/gid = 1
-	var/id = 0
+	id = 0
```

Comment:
```suggestion
```

bad idea/unnecessary?

Reply:
these are used by scrubbers internally

Reply:
This is set by a global counter. The default value can be removed.